operations/validate_csm_health.md

```diff
@@ -603,39 +675,69 @@ This procedure must run on a master or worker node (**not the PIT node and not `
    ```
 
 1. Log out from the UAI
+
    ```bash
    vers@uai-vers-a00fb46b-6889b666db-4dfvn:~> exit
    ncn#
    ```
 
 1. Clean up the UAI.
+
    ```bash
    ncn# cray uas delete --uai-list $UAINAME
    ```
 
    Expected output looks similar to the following:
+
    ```text
    results = [ "Successfully deleted uai-vers-a00fb46b",]
    ```
 
 If the commands ran with similar results, then the basic functionality of the UAS and UAI is working.
 
+### 6.3 Test UAI Gateway Health
+
+Like the NCN gateway health check, the gateway tests check the health of the API Gateway on all of the relevant networks.   
+On UAIs, the API gateway should only be accessible on the user network (either CAN or CHN depending on the configuration of the system). 
+The gateway tests will check that the gateway is accessible on all networks it should be accessible and NOT accessible on all 
+networks it should NOT be accessible.  It will also check several service endpoints to verify that they return the proper response 
```

Comment:
```suggestion
networks it should NOT be accessible. It will also check several service endpoints to verify that they return the proper response 
```